src/stdune/ansi_color.ml

```diff
@@ -136,7 +136,20 @@ module Styles = struct
     Style.escape_sequence l
 end
 
-module Render = Pp.Renderer.Make(struct
+let term_supports_color = lazy (
```

Comment:
It looks like there is very similar code in src/colors.ml. Probably one of the versions should be removed.

Reply:
Indeed, I removed it